src/parser/opencypher/parser.hpp

```diff
@@ -14,17 +14,29 @@
 #include <string>
 
 #include "antlr4-runtime.h"
-#include "query/v2/exceptions.hpp"
-#include "query/v2/frontend/opencypher/generated/MemgraphCypher.h"
-#include "query/v2/frontend/opencypher/generated/MemgraphCypherLexer.h"
+#include "utils/exceptions.hpp"
+#include "parser/opencypher/generated/MemgraphCypher.h"
+#include "parser/opencypher/generated/MemgraphCypherLexer.h"
```

Comment:
The `src/query/frontend/stripped_lexer_constants.hpp` file is mentioned in the lexers (both Cypher and Memgraph). Doesn't it should belong to this library then?